yogstation/code/modules/donor/unique_donator_items.dm

```diff
@@ -74,19 +78,45 @@ GLOBAL_DATUM_INIT(donator_gear, /datum/donator_gear_resources, new)
 			qdel(S)
 			continue
 		donor_items += S
+	for(var/Plushtype in subtypesof(/obj/item/toy/plush)) //generate our plushies
+		var/datum/donator_gear/P = new
+		var/obj/item/toy/plush/myplush = new Plushtype
```

Comment:
Don’t spawn a plushie, use initial() to access vars

Reply:
You’re doing it wrong then

Reply:
Change
var/obj/item/toy/plush/myplush = new Plushtype
to var/obj/item/toy/plush/myplush = Plushtype

Change all mentions of myplush.variable to initial(myplush.variable)